save-payment-method/server/views/checkout.ejs

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PayPal JS SDK Save Payment Method Integration</title>
+  </head>
+  <body>
+    <div id="paypal-button-container"></div>
+    <p id="result-message"></p>
+    <script
+      src="https://www.paypal.com/sdk/js?client-id=<%= clientId %>"
```

Comment:
I'm also seeing this warning in the console: 
<img width="775" alt="Screenshot 2023-11-03 at 08 51 47" src="https://github.com/paypal-examples/docs-examples/assets/3824954/17f61ede-4278-4f6e-a7ae-5ddb8492a64b">

though the docs do not include `vault=true` in the [code sample](https://developer.paypal.com/docs/checkout/save-payment-methods/during-purchase/js-sdk/paypal/#link-passuseridtokentojavascriptsdk)

Reply:
I added it in d4ea5c3 and the warning went away 🥳